packages/wonder-blocks-core/components/clickable-behavior.test.js

```diff
@@ -11,6 +12,18 @@ const keyCodes = {
 };
 
 describe("ClickableBehavior", () => {
+    // Note: window.location.assign needs a mock function in the testing
+    // environment.
+    window.location.assign = jest.fn();
+
+    beforeEach(() => {
+        unmountAll();
+    });
+
+    afterEach(() => {
+        window.location.assign.mockClear();
```

Comment:
Because this is a global, need to reset it because one of the tests test how many times this was called.